assets/src/components/button/style.scss

```diff
@@ -0,0 +1,83 @@
+/**
+ * Button styles,
+ */
+
+
+.newspack-button {
+	.components-button {
+	    padding: 0 20px 1px;
+		text-align: center;
+
+		font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", sans-serif;
+		font-size: 14px;
+		line-height: 18px;
+		font-weight: 500;
+		color: $muriel-gray-700;
+		text-shadow: none;
```

Comment:
After removing the wrapper `div`, `text-shadow` will need to move into the `&.is-button` block.

Reply:
That will make the shadow appear on the default button style, too, which I don't think we really want to do